.github/workflows/bindings.yml

```
@@ -13,7 +13,7 @@ env:
   CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
   CIBW_BUILD: cp3?-*
   CIBW_ARCHS_MACOS: "x86_64"
```

```suggestion
  CIBW_ARCHS_MACOS: "x86_64 arm64"
```
Don't you want to add support for apple silicon wheels?